src/components/TextInput/BaseTextInput.js

```diff
@@ -220,7 +220,7 @@ class BaseTextInput extends Component {
                     </TouchableWithoutFeedback>
                 </View>
                 {!_.isEmpty(this.props.errorText) && (
-                    <InlineErrorText>
+                    <InlineErrorText additionalStyles={hasLabel ? [styles.ml2] : []}>
```

Comment:
No need to put `styles.ml2` in an array, and the falsy value should be an empty object.